src/Symfony/Component/Notifier/Bridge/Octopush/README.md

```diff
@@ -0,0 +1,25 @@
+Octopush Notifier
+===============
+
+Provides [Octopush](https://www.octopush.com) integration for Symfony Notifier.
```

Comment:
```suggestion
Provides [Octopush](https://octopush.com) integration for Symfony Notifier.
```

Reply:
As stated above. The SSL certificate is only valid for the 'www' domain. I can change the url without 'www' but there will always be a redirection.